services/simcore/docker-compose.yml

```diff
@@ -77,6 +77,10 @@ services:
         delay: 5s
         max_attempts: 3
         window: 120s
+      placement:
+        constraints:
+          - node.role == worker
+          - node.labels.dynamicsidecar == true
```

Comment:
Question: why is ```node.labels.dynamicsidecar == true``` not already sufficient?
The agent shall go everywhere, where a dynamic sidecar may go. I am not sure that constraining to worker is useful. it might even create issues if at any moment in time you have a manager that runs a sidecar for whatever reason (even if remote)

Reply:
I deleted `role == worker` constraint